std_msgs/Header header
string uuid
string roi
int64 start_time
relational_learner/episodeMsg[] episodes
